app/controllers/notes_controller.rb

```diff
@@ -17,6 +17,7 @@ def update
   end
 
   def index
+    @notes = Note.find(current_user)
```

Comment:
here are a couple ways this can be done...
`@notes = current_user.notes`
or
`@notes = Note.where(user: current_user)`